### DISCUSSION TITLE
Nothing here works now

### DISCUSSION THREAD

**Discussion by duq1** [3kh0/ext-remover#483] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello, 
My school uses lightspeed filter agent and many of the things here no longer work. webstorex is blocked by my school, inspect element has been blocked, and I am unable to use bookmarklets. Tinyurl is also blocked. Killcurly used to work, but my account can no longer be unsynced.  CORRECTION: chrome account can be unsynced, but sites are still blocked. If there is some other method, please do tell.

**Comment by S-PScripts** [comment, marked as answer]
--
